src/gurt/hash.c

```diff
@@ -947,6 +947,9 @@ d_hash_table_destroy_inplace(struct d_hash_table *htable, bool force)
 	uint32_t		 i;
 	int			 rc = 0;
 
+	if (htable->ht_buckets == NULL)
+		return 0;
```

Comment:
What's the change for?

Reply:
oh, a bug that hash table might be freed twice bug when i tried to use multiple DBs, but i found multiple DBs could not work, then switch back, but did not remove it.